packages/react/src/components/navigation/navigationUser/NavigationUser.tsx

```diff
@@ -34,49 +35,57 @@ export const NavigationUser = ({
   userName,
   ...dropdownProps
 }: NavigationUserProps) => {
-  const { dispatch, isMobile } = useContext(NavigationContext);
+  const { dispatch } = useContext(NavigationContext);
 
   // dispatch auth state
   useEffect(() => dispatch({ type: 'AUTHENTICATED', value: authenticated }), [authenticated, dispatch]);
 
-  const userItems: React.ReactNode = isMobile ? (
+  const userItems = (
     <>
-      {userName && (
-        <span className={styles.userName}>
-          <IconUser aria-hidden />
-          {userName}
-        </span>
-      )}
-      {children}
+      <Visible above="m">
```

Comment:
Think this should be on mobile, so below="m"? Also row 54 should be on desktop, so above="m" there.

Reply:
fixed